### DISCUSSION TITLE
After desktop environment hopping, lxqt can't start

### DISCUSSION THREAD

**Discussion by QiangF** [lxqt/lxqt#2599] [Q&A]
The whole journal file is here https://0x0.st/X4lq.txt

**Comment by stefonarch** [comment]
Some more details about which system, which Wms and so on wouldn't hurt at all.

**Reply by QiangF** [reply]
Arch linux with herbstluftwm. I tried xfce for a while, and then switched to lxqt. Booting a stand alone herbstluftwm session works, but all qt applications can't start, including my favorite image viewer nomacs. 

    (pip) q@mypc:~$ loginctl session-status
4 - q (1000)
  Since: Sat 2024-08-17 22:23:43 CST; 3min 44s ago
  State: active
 Leader: 3214 (lightdm)
   Seat: seat0; vc7
    TTY: tty7
 Remote: no
Service: lightdm
   Type: x11
  Class: user
Desktop: herbstluftwm
   Idle: no
   Unit: session-4.scope
         ├─ 3214 lightdm --session-child 12 20
         ├─ 7702 dbus-run-session -- herbstluftwm --locked
         ├─ 7758 dbus-daemon --nofork --print-address 4 --session
         ├─ 7759 herbstluftwm --locked
         ├─ 7760 /usr/bin/bash /home/q/.config/herbstluftwm/autostart
         ├─ 7957 systemctl --user start x@:0.target
         ├─ 7961 herbstclient -i "(focus_changed|reload|tag_changed)"
         ├─ 7962 /usr/bin/bash /home/q/.config/herbstluftwm/autostart
         ├─ 7978 /usr/bin/dunst
         ├─ 7991 /usr/lib/gvfsd
         ├─ 8014 /usr/lib/at-spi-bus-launcher
         ├─ 8020 /usr/bin/dbus-daemon --config-file=/usr/share/defaults/at-spi2/accessibility.conf --nofork --print-address 11 --address=unix:path=/run/user/1000/at-spi/bus_0
         ├─ 8039 /usr/lib/xdg-desktop-portal
         ├─ 8057 /usr/lib/xdg-permission-store
         ├─ 8069 /usr/lib/at-spi2-registryd --use-gnome-session
         ├─ 8071 /usr/lib/xdg-desktop-portal-gtk
         ├─ 9904 /usr/lib/gvfsd-trash --spawner :1.8 /org/gtk/gvfs/exec_spaw/0
         ├─ 9917 /usr/lib/gvfs-udisks2-volume-monitor
         ├─10133 /usr/lib/firefox-esr/firefox-esr

**Comment by stefonarch** [comment]
No idea what happens at your side, installed herbstluftwm and chose it in Session Settings, no issue:

![immagine](https://github.com/user-attachments/assets/1b22ffc8-578b-4101-b578-7c04107e258d)

**Reply by QiangF** [reply]
I guess I need a fresh installation. Thank you for trying herbstluftwm.

**Reply by stefonarch** [reply]
Maybe only testing a new user is enough, default values of everything.
